packages/environment-check/src/checks/get-installed.ts

```diff
@@ -159,6 +160,12 @@ export async function getFioriGenVersion(): Promise<string> {
  * @returns modules and versions
  */
 export async function getProcessVersions(): Promise<NodeJS.ProcessVersions> {
```

Comment:
```suggestion
export async function getProcessVersions(logger?: ReturnType<typeof getLogger>): Promise<NodeJS.ProcessVersions> {
```

Instead of `ReturnType<typeof getLogger>` you could also define a logger interface in `types.ts` that the class `Logger` implements, but you don't have to.

Reply:
Implemented the ILogger interface